pymc3/tests/test_distributions.py

```diff
@@ -2933,6 +2942,48 @@ def test_car_logp(size):
     assert np.allclose(delta_logp - delta_logp[0], 0.0)
 
 
+@pytest.mark.parametrize("sparse", [True, False])
+def test_car_rng_fn(sparse):
```

Comment:
This would probably be better placed in `test_distributions_random.py`

Reply:
I moved it into `test_distributions_random.py`.